FHIR-Converter/client/src/view/common/commands/command-handler.ts

```diff
@@ -19,6 +28,6 @@ export async function commandHandler(event) {
 		await this(event);
 	} catch (error) {
 		// Handle the error
-		errorHandler.handle(error);
+		errorHandler.handle(error, this);
```

Comment:
why do we need add 'this' here? just wrap the error into ConversionError or CommandError?

Reply:
In order to record which command triggered the error.